translations/terminal_tutorial/terminal_tutorial.kor.md

```diff
@@ -0,0 +1,375 @@
+
+---
+
+> ## **공지:**
+>
+> 이 프로젝트의 메인테이너가 되어 도움을 주고 싶으신가요? 관심이 있으시다면 메인테이너 가이드 [maintainer's guide](/maintainer_guide.md) 를 읽고 [Twitter](https://twitter.com/Syknapse) 에서 DM을 보내주세요.
+
+---
+
+### 빠른 액세스 인덱스
+
+#### 개요
+
+- [공지](#공지)
+- [소개](#소개)
+- [누구를 위한 프로젝트인가요?](#누구를-위한-프로젝트인가요?)
+- [이 프로젝트는 왜 필요한가요?](#이-프로젝트는-왜-필요한가요?)
+- [무엇을 기여하게 되나요?](#무엇을-기여하게-되나요?)
+- [번역](#번역)
+- [준비](#준비-)
+- [다음 단계](#다음-단계)
+- [감사의 말](#감사의-말)
+
+#### 진행단계
+
+- [기여하기](#기여하기)
+- [1단계 - 저장소 포크(fork)하기](#1단계-저장소-포크fork하기)
+- [2단계 - 저장소 복제(clone)하기](#2단계-저장소-복제clone하기)
+- [3단계 - 새 브랜치 생성](#3단계-새-브랜치-생성)
+- [4단계 - 메인 index.html 파일 열기](#4단계-메인-indexhtml-파일-열기)
+- [5단계 - 카드 템플릿 복사하기](#5단계-카드-템플릿-복사하기)
+- [6단계 - 내용 변경 적용하기](#6단계-내용-변경-적용하기)
+- [7단계 - 변경사항 커밋(commit)하기](#7단계-변경사항-커밋commit하기)
+- [8단계 - Github에 푸시(push)하기](#8단계-github에-푸시push하기)
+- [9단계 - PR 제출하기](#9단계-prpull-request-제출하기)
+- [10단계 - 축하합니다](#10단계-축하합니다)
+
+---
+
+## 소개
+
+이곳에선 이 프로젝트 또는 GitHub/Git에 _터미널_ 을 사용하여 기여하는 방법을 배우게 됩니다.
+기여가 처음인 분들이 쉽고 간단하게 프로젝트에 참여할 수 있도록 도움을 주기 위한 튜토리얼입니다.
+
+### 목표
+
+- 오픈소스 프로젝트에 기여하기.
+- Github를 더 능숙하게 사용하기.
+
+### 누구를 위한 프로젝트인가요?
+
+- 이 프로젝트는 완전 초보자들을 위한 프로젝트입니다. 앵커 태그 `<a href="" target=""></a>`를 작성하고 수정할 수 있다면 진행 할 수 있습니다.
+- 처음으로 오픈소스에 기여 해보고 싶거나, 이미 경험이 있지만 더 많은 기여를 경험하고 자신감을 얻고 싶은 분들을 위한 프로젝트입니다.
+
+### 이 프로젝트는 왜 필요한가요?
+
+웹 개발자 취업을 희망하는 사람들과 현직 개발자들에게 Git 버전 관리는 필수입니다. GitHub는 많은 사람들이 사용하는 가장 대중적인 Git 호스팅 서비스이자 오픈소스 커뮤니티입니다. GitHub를 능숙하게 사용하는 것은 개발자로서 필수이며 중요한 기술 중 하나입니다. 프로젝트에 기여하면 GitHub 사용에 대한 자신감이 높아지고 GitHub 프로필에 기여 내용이 표시됩니다.
+
+만약 본인이 입문자이며 Git과 GitHub에 대해 배워야 할지 고민을 하고 있다면 여기에 답이 있습니다: [You Should've Learned Git Yesterday](https://codeburst.io/number-one-piece-of-advice-for-new-developers-ddd08abc8bfa 'New Developer? You should’ve learned Git yesterday. by Brandon Morelli, creator of CodeBurst.io').
+
+### 무엇을 기여하게 되나요?
+
+![Contributor Card](/readme-only/card.PNG 'Contributor Card')
+
+여러분은 이 [프로젝트의 웹 페이지](https://syknapse.github.io/Contribute-To-This-Project/ 'https://syknapse.github.io/Contribute-To-This-Project')에 위 그림과 같은 카드를 기여하게 될 것입니다. 카드에는 본인 이름, 트위터 아이디, 간단한 소개 및 여러분들이 추천하는 웹 개발자들을 위한 유용한 리소스 링크가 포함됩니다.
+
+HTML 파일 내부에 카드 템플릿의 복사본을 만들고 본인의 정보로 수정하세요.
+
+---
+
+### 번역
+
+이 튜토리얼은 [다른 언어](/translations/README.md)로도 제공됩니다.
+
+| [English](/terminal_tutorial.md) | [Bangla](/translations/terminal_tutorial/terminal_tutorial.ben.md) | [Chinese (Traditional)](/translations/terminal_tutorial/terminal_tutorial.zho-tc.md) | [Hindi](/translations/terminal_tutorial/terminal_tutorial.hin.md) | [Portuguese (Brazilian)](/translations/terminal_tutorial/terminal_tutorial.por-br.md) |
+| :----: | :----: | :----: | :----: | :----: |
+| [Ukrainian](/translations/terminal_tutorial/terminal_tutorial.ukr.md) |
+
+> 프로젝트 문서 번역을 하고 싶다면 환영입니다. [`번역 가이드`](/translations/README.md)를 읽고 기여해주세요.
+
+---
+
+### 준비! :)
+
+참고: 만약 커맨드 라인 환경이 익숙하지 않다면 [GitHub Desktop을 사용하는 GUI 튜토리얼](/README.md)을 봐주세요. 
```

Comment:
this link should forward to korean README translation

Reply:
Updated the link